GNUmakefile

```diff
@@ -63,18 +63,19 @@ chainlink-test: ## Build a test build of chainlink binary.
 
 .PHONY: install-loopinstall
 install-loopinstall:
+#TODO go tool?
 	go install github.com/smartcontractkit/chainlink-common/pkg/loop/cmd/loopinstall
 
 .PHONY: install-plugins-public
-install-plugins-public: ## Build & install public remote LOOPP binaries (plugins).
+install-plugins-public: install-loopinstall ## Build & install public remote LOOPP binaries (plugins).
```

Comment:
The reason why I didn't make `install-loopinstall` dependent here is in the Dockerfile we want to specify a different GOBIN path for loopinstall itself so it doesn't get copied into the same directory as the plugins. I would prefer to do this but if we do, we need to make sure that `loopinstall` isn't in the final image.